assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -283,6 +340,40 @@ const baseSelectors = {
 
 		return dismissedTourSlugs.includes( slug );
 	} ),
+
+	/**
+	 * Gets the timestamp for the last dismissed feature tour.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {number} Timestamp of the last dismissal.
```

Comment:
This may also return `undefined` if the value has not been resolved yet, or if there is no entry in the cache yet.